palindrome.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <string.h>
```

Comment:
you also didn't properly use a function to make your program! you didn't use recursion.